customer-service/build.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
 
     compile "org.springframework.boot:spring-boot-starter-actuator:$springBootVersion"
     compile "org.springframework.boot:spring-boot-starter-web:$springBootVersion"
+    compile group: 'org.springframework.hateoas', name: 'spring-hateoas', version: '0.25.0.RELEASE'
```

Comment:
Why Hateos was added?

Reply:
Chris, this PR is based on other PR which contains many updating commits.
As I remember, I added hateoas because there were some issues with dependencies.
Spring produced java.lang.NoClassDefFoundError or something.
But hateoas solved it.
I will check if it still necessary.

Reply:
But if hateoas API is not used then it doesn't make sense to depend on it. It probably has the correct version of a dependency that fixes the problem.

Reply:
I will check